src/models/room.ts

```diff
@@ -2928,14 +2955,20 @@ export class Room extends TypedEventEmitter<EmittedEvents, RoomEventHandlerMap>
             // check for an alias, if any. for now, assume first alias is the
             // official one.
             const mRoomName = this.currentState.getStateEvents(EventType.RoomName, "");
-            if (mRoomName && mRoomName.getContent() && mRoomName.getContent().name) {
-                return mRoomName.getContent().name;
+            if (mRoomName?.getContent().name) {
```

Comment:
The previous code did the equivalent of 
```suggestion
            if (mRoomName?.getContent()?.name) {
```
are you sure that getContent() always returns an object? We don’t have strict null checking yet in js-sdk.

Reply:
![image](https://user-images.githubusercontent.com/2403652/185928181-eab83668-2196-42b0-804d-5aacb08fb64e.png)
![image](https://user-images.githubusercontent.com/2403652/185928212-0b3c4351-bbdc-496d-b2f2-d82d211482df.png)

It doesn't always return an object, `content` or `m.new_content` could be a boolean, because we don't do runtime type checking on incoming events, but it does the `|| {}` that this was effectively doing already